eox_core/api/v1/serializers.py

```diff
@@ -257,7 +257,7 @@ class EdxappGradingPolicySerializer(serializers.Serializer):
     """
     Serializes the course grading policy
     """
-    grader = EdxappGradingRawPolicySerializer(many=True, required=False, source="GRADER")
+    grader = EdxappGradingRawPolicySerializer(many=True, source="GRADER")
```

Comment:
why is it required now?

Reply:
Because I realized that when you are retrieving a grading policy it always includes the `grader` and `grade_cutoffs` keys. The whole object `grading_policy` is what is optional in the response.